docs/installation/configuration-file/_index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Configuration File"
+linkTitle: "Configuration File"
+weight: 30
+no_list: true
+type: docs
+draft: false
+icon: "/installation/img/thumbnails/manage.png"
+images: ["/installation/img/thumbnails/manage.png"]
+description: "Building your robot configuration file for use with viam-server."
+---
+
+The `viam-server` binary uses a JSON-formatted configuration file to define all resources (hardware [components](/components/) and software [services](/services/)) it has access to, as well as any relevant parameters for those resources.
+
+In order to start `viam-server` on a {{< glossary_tooltip term_id="board" text="board" >}} or computer, you must have a valid configuration file present on the local system.
+
+For more information on the individual configuration options available, see [Configuration](/manage/configuration/).
+
+## Configuration file location
+
+By default, the configuration file is located at:
+
+* Linux: <file>/etc/viam.json</file>
+* macOS: <file>/opt/homebrew/etc/viam.json</file>

```

Comment:
These are really only "default" for the system service (systemd or brew service on mac) The viam-server binary itself has no default config location, so it must ALWAYS be specified when running directly.